front/src/views/user/user-settings/user-settings.view.js

```diff
@@ -0,0 +1,73 @@
+import { MmtTextInput, MmtSelect, MmtSubmitButton } from '@components';
+import axios from 'axios';
+const { serverPath, emailRegex } = require('@/config/config.dev.json');
+
+export default {
+  components: {
+    MmtTextInput,
+    MmtSelect,
+    MmtSubmitButton
+  },
+  data () {
+    return {
+      genders: [],
+      languages: [],
+      countries: [],
+      lastName: '',
+      firstName: '',
+      email: '',
+      gender: '',
+      language: '',
```

Comment:
is these one connected whith localization? Should be

Reply:
These fields are containing ids, translation generates here: convertToSelectCompatible()

Reply:
yes, as I said it may make feeling of flexibility, but from other side it makes code more difficult for reading and support